doc/nrf/device_guides/working_with_nrf/nrf70/developing/debugging.rst

```diff
@@ -0,0 +1,43 @@
+.. _ug_nrf70_developing_debugging:
+
+Debugging
+#########
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide describes how to debug applications that use the nRF70 Series companion chips. It covers the following topics:
```

Comment:
Perhaps it is not required, as we do not categorize the document based on these headings.
```suggestion
This guide describes how to debug applications that use the nRF70 Series companion chips.